lib/src/Modal/Modal.tsx

```diff
@@ -23,21 +24,45 @@ export type ModalProps = JSX.IntrinsicElements["div"] & {
     position?: ModalPosition;
     /** Modal toggle */
     toggle?: boolean;
-    /** To only allow tab to focus within modal */
-    trapFocus?: boolean;
```

Comment:
before removing it entirely, should we just mark it @deprecated and keep it until next major release?

Reply:
if I've added in a breaking change commit 57d231c1421c00e4b4946f2322d7bd7530a92e76 while working on one of the accessibility changes, would this bump our major version while preping the next release?

Reply:
after discussion with the team, the `trapFocus` prop would be deprecated regardless if the major version were bumped